app/src/main/java/ca/bc/gov/bchealth/ui/login/BcscAuthFragment.kt

```diff
@@ -0,0 +1,151 @@
+package ca.bc.gov.bchealth.ui.login
+
+import android.app.Activity
+import android.content.Intent
+import android.os.Bundle
+import android.text.Html
+import android.view.View
+import androidx.activity.result.ActivityResult
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.core.view.isVisible
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import androidx.lifecycle.Lifecycle
+import androidx.lifecycle.SavedStateHandle
+import androidx.lifecycle.lifecycleScope
+import androidx.lifecycle.repeatOnLifecycle
+import androidx.navigation.fragment.findNavController
+import ca.bc.gov.bchealth.R
+import ca.bc.gov.bchealth.databinding.FragmentBcscAuthBinding
+import ca.bc.gov.bchealth.utils.showError
+import ca.bc.gov.bchealth.utils.viewBindings
+import com.google.android.material.dialog.MaterialAlertDialogBuilder
+import dagger.hilt.android.AndroidEntryPoint
+import kotlinx.coroutines.launch
+
+/*
+* @auther amit_metri on 04,January,2022
+*/
+@AndroidEntryPoint
+class BcscAuthFragment : Fragment(R.layout.fragment_bcsc_auth) {
+
+    private val binding by viewBindings(FragmentBcscAuthBinding::bind)
+    private val viewModel: BcscAuthViewModel by viewModels()
+    private val authResultLauncher = registerForActivityResult(
+        ActivityResultContracts.StartActivityForResult()
+    ) { activityResult ->
+        processAuthResponse(activityResult)
+    }
+    private lateinit var savedStateHandle: SavedStateHandle
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        savedStateHandle = findNavController().previousBackStackEntry!!.savedStateHandle
```

Comment:
call it directly like
findNavController().previousBackStackEntry?.savedStateHandle?.set(BCSC_AUTH_SUCCESS, false)

Reply:
done